karabo_data/reader.py

```diff
@@ -634,39 +662,19 @@ def get_array(self, source, key, extra_dims=None, roi=by_index[...]):
 
         seq_arrays = []
 
-        if source in self.control_sources:
-            data_path = "/CONTROL/{}/{}".format(source, key.replace('.', '/'))
-            for f in self._source_index[source]:
-                slices = (slice(None, len(f.train_ids)),) + roi
-                data = f.file[data_path][slices]
-                if extra_dims is None:
-                    extra_dims = ['dim_%d' % i for i in range(data.ndim - 1)]
-                dims = ['trainId'] + extra_dims
+        for chunk in self._find_data_chunks(source, key):
+            print("Got chunk in", chunk.file)
```

Comment:
is this a debug print that snuck under the radar?

Reply:
It is, good catch! Just removed it: 9de53011d2560090d175d8fb62570bb25eb7acd9